compute/rn/release-information/release-notes-22-12-update1.adoc

```diff
@@ -0,0 +1,93 @@
+:toc: macro
+== 22.12 Update 1 Release Notes
+
+The following table provides the release details:
+
+[cols="1,4"]
+|===
+|Build
+|22.12.xxx
+
+|Codename
+|Lagrange, 22.12 Update 1
+|Release date
+|Jan 9, 2023
+
+|Type
+|Maintenance release
+
+|SHA-256 digest
+|TBD
+|===
+
+//Besides hosting the download on the Palo Alto Networks Customer Support Portal, we also support programmatic download (e.g., curl, wget) of the release directly from our CDN:
+
+// LINK
+
+toc::[]
+
+=== New Features in Core
+// CWP-42365
+==== Filter Defender /api/vVERSION/defenders by TAS Foundation ID
+Introduces a new value,*tasFoundations*, in the *fields* query parameter to filter Defenders by TAS Foundation ID.
+  
+//CWP-41281
+==== ARM64 bit twistcli for MacOS platforms
+Download ARM64 Mac compatible version of twistcli under *Manage > System > Utilities*, or via */util/osx/arm64/twistcli* API.
+
+//CWP-42198 - Rodrigo's validation needed
+==== Support for Talos Linux - container vulnerabilities and compliance
+
+Talos Linux is now supported as a host OS for orchestrator container Defenders.
+Defenders installed on Talos Linux will allow scanning vulnerability and compliance for the running containers, as well as performing registry scans.
+To deploy on Talos Linux cluster, use the new "Talos Linux deployment" toggle in the Defenders deployment page, or the new `--talos` flag in twistcli.
```

Comment:
@divyabhushan @rcaballeromx 
We have some limitations for the TalOS support, can we please make sure it is described in the system requirements where we declare support for Talos? The limitations are described in the epic under "Docs guidelines" https://redlock.atlassian.net/browse/CWP-42198

Reply:
@rcaballeromx is adding it to the known issues, I believe.